region/lowernorfair/west.json

```diff
@@ -841,7 +840,16 @@
           "to": [
             {
               "id": 2,
-              "requires": [ "canOpenGreenDoors" ],
+              "requires": [
```

Comment:
I'm a bit surprised there's no fast ripper hits in there. Sounds unlikely without killing them.

Reply:
We removed the entire part about killed enemies since you didn't like it, remember? I killed two rippers with supers for node 1 to 2 since supers are already required to open the gate anyway and any ammo concerns are mute since rippers drop the supers en masse. 

For node 2 to 1 however I could get through without killing or get hit by any. While that strat was harder to execute than any of the other plausible ones like using PB's or ScrewAttack they where not too far off each-other in the use of heatFrames, The rippers don't deal much damage anyway if you have to get past this sloppy style. The bulk of the damage for most runners will come from executing the GGG so I simply opted for a flat heatFrame count in this link as well.